### PR TITLE
Fix homepage vendors

### LINKED ISSUE #9554: Totals are wrong on overview page
The numbers next to the links of the vendors and releases on the overview page are wrong, they aren't filtered correctly.

### PR DESCRIPTION
**Pull request by carkod**
## Done

Correction on number of vendors and releases

## QA

- go to https://ubuntu-com-9608.demos.haus/certification and check that the numbers are correct (not sure how to check the accuracy, but as far as I know Desktop = Laptop and desktops, and Iot = smart core so it should be correct)

## Issue / Card

Fixes #9554

### CODE REVIEW

**Comment by bethcollins92** [comment]
The numbers are different on the results page for some of the links. If you look at "Huawei Technologies Co., Ltd. (70)" from the SoC section on the overview page, the actual results on the search results page are only 6, but the number in the brackets says 70. This discrepancy is the same for most of the numbers

**Comment by carkod** [comment]
> The numbers are different on the results page for some of the links. If you look at "Huawei Technologies Co., Ltd. (70)" from the SoC section on the overview page, the actual results on the search results page are only 6, but the number in the brackets says 70. This discrepancy is the same for most of the numbers. (I've fixed the link in another [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/9619) from the desktop section to also include laptops which I think should fix the incorrect numbers for that section)

Thanks. Fixed

**Comment by bethcollins92** [comment]
> > The numbers are different on the results page for some of the links. If you look at "Huawei Technologies Co., Ltd. (70)" from the SoC section on the overview page, the actual results on the search results page are only 6, but the number in the brackets says 70. This discrepancy is the same for most of the numbers. (I've fixed the link in another [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/9619) from the desktop section to also include laptops which I think should fix the incorrect numbers for that section)
> 
> Thanks. Fixed

The numbers are still different on the "Ubuntu Server certified hardware" section. Also, do you think we should include `form=Laptop` in the urls in the "Ubuntu Desktop certified hardware" section? otherwise laptops aren't included on any of the links. That's what I have changed in the [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/9619) mentioned above,  but can revert it.

**Comment by carkod** [comment]
> > > The numbers are different on the results page for some of the links. If you look at "Huawei Technologies Co., Ltd. (70)" from the SoC section on the overview page, the actual results on the search results page are only 6, but the number in the brackets says 70. This discrepancy is the same for most of the numbers. (I've fixed the link in another [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/9619) from the desktop section to also include laptops which I think should fix the incorrect numbers for that section)
> > 
> > 
> > Thanks. Fixed
> 
> The numbers are still different on the "Ubuntu Server certified hardware" section. Also, do you think we should include `form=Laptop` in the urls in the "Ubuntu Desktop certified hardware" section? otherwise laptops aren't included on any of the links. That's what I have changed in the [PR](https://github.com/canonical-web-and-design/ubuntu.com/pull/9619) mentioned above, but can revert it.

The releases number should match now, I reverted to the previous API, I wonder why we were using the summaryvendor API in the current site, it seems to give different numbers.
Also, looking at current site, Desktop and Laptop seem to be in the same category so I also did it here.

File: webapp/certification/views.py [hunk 1]
```diff
@@ -220,7 +223,9 @@ def certification_home():
             all_releases = sorted(all_releases, reverse=True)
 
         if int(release["desktops"]) > 0 or int(release["laptops"]) > 0:
-            release["path"] = f"/certification?form=Desktop&release={version}"
+            release[
+                "path"
+            ] = f"/certification?form=Desktop&form=Laptop&release={version}"
             desktop_releases.append(release)
```
Comment (carkod) [inline webapp/certification/views.py:229]: This is python linting

File: webapp/certification/views.py [hunk 2]
```diff
@@ -235,6 +240,12 @@ def certification_home():
             ] = f"/certification?form=Server%20SoC&release={version}"
             soc_releases.append(release)
 
+        if int(release["servers"] > 1):
+            release[
+                "path"
+            ] = f"/certification?form=Server%20SoC&release={version}"
+            server_releases.append(release)
+
```
Comment (carkod) [inline webapp/certification/views.py:248]: Switched to old API endpoint, the one the rest is using

**Comment by bethcollins92** [comment]
Thanks @carkod - QA LGTM 👍🏼

**Review by codersquid — APPROVED** [review]
I'm glad you stopped using the summary endpoint. it is expensive, and it's getting information that you don't need.

**Pull request merged**